quickstarts/library-process/pom.xml

```diff
@@ -28,8 +28,8 @@
 			resources, i.e. build is platform dependent! -->
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.compiler.version>3.8.0</maven.compiler.version>
-		<ba-platform-bom.version>7.8.1.redhat-00002</ba-platform-bom.version>
-		<org.jboss.bom.version>7.3.2.GA</org.jboss.bom.version>
+		<ba-platform-bom.version>7.10.1.redhat-00006</ba-platform-bom.version>
```

Comment:
Same as the other PR, this should be 7.10.1.redhat-00001

Reply:
updated, thanks